src/pam_types.rs

```diff
@@ -23,7 +24,7 @@ pub enum PamMsgStyle {
 #[repr(C)]
 pub struct PamMessage {
```

Comment:
Self-note: this comes from https://github.com/linux-pam/linux-pam/blob/b429ea18b1c9c8953df5169c6a453b4255a6f23d/libpam/include/security/_pam_types.h#L244-L251.